src/Datadog.Trace.BenchmarkDotNet/DatadogExporter.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using Datadog.Trace.Ci;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.BenchmarkDotNet
+{
+    /// <summary>
+    /// Datadog BenchmarkDotNet Exporter
+    /// </summary>
+    public class DatadogExporter : IExporter
+    {
+        /// <summary>
+        /// Default DatadogExporter instance
+        /// </summary>
+        public static readonly IExporter Default = new DatadogExporter();
+
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(DatadogExporter));
+
+        static DatadogExporter()
+        {
+            // Preload environment variables.
+            CIEnvironmentValues.DecorateSpan(null);
+        }
+
+        /// <inheritdoc />
+        public string Name => nameof(DatadogExporter);
+
+        /// <inheritdoc />
+        public void ExportToLog(Summary summary, ILogger logger)
+        {
+        }
+
+        /// <inheritdoc />
+        public IEnumerable<string> ExportToFiles(Summary summary, ILogger consoleLogger)
+        {
+            DateTimeOffset startTime = DateTimeOffset.UtcNow;
+            Exception exception = null;
+
+            try
+            {
+                Tracer tracer = new Tracer();
+
+                foreach (var report in summary.Reports)
+                {
+                    Span span = tracer.StartSpan("benchmark.test", startTime: startTime);
```

Comment:
If I'm understanding this correctly, each report will start a new Trace with only one span inside of it, right? I just want to make sure I understand the output we're generating here

Reply:
That's right, there is not much traces in a benchmark test. So we add a single span representing the test itself with all the data as tags. This will be correlated later with CI + GIT info.

Reply:
Just by looking the line of code you comment it, I have to change the operation name with `benchmarkdotnet.test` to follow the semantic conventions. I'll do the change.

Reply:
done in: https://github.com/DataDog/dd-trace-dotnet/pull/817/commits/b7a65c5dbc2799ffca6e53b853932d5f2dd88f86